### ISSUE TITLE
Failed to load the script-runner package in Ubuntu 16.04 and Atom 1.23.1 x64

### ISSUE THREAD

**Issue by rohan-paul** [ioquatix/script-runner#85]
After installing from within Atom I get this error - ``Failed to load the script-runner package``

The complete stack trace

```
**Atom**: 1.23.1 x64
**Electron**: 1.6.15
**OS**: Ubuntu 16.04.3
**Thrown From**: [script-runner](https://github.com/ioquatix/script-runner) package 2.2.1

### Stack Trace

Failed to load the script-runner package

At Terminal.loadAddon is not a function

TypeError: Terminal.loadAddon is not a function
    at /packages/script-runner/lib/script-runner-view.js:9:10)
    at /packages/script-runner/lib/script-runner-view.js:180:42)
    at Module.get_Module._compile (/usr/share/atom/resources/app/src/native-compile-cache.js:106:36)
    at Object.value [as .js] (/usr/share/atom/resources/app/src/compile-cache.js:239:29)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (/app.asar/static/index.js:47:45)
    at require (/usr/share/atom/resources/app/src/native-compile-cache.js:66:33)
    at /packages/script-runner/lib/script-runner.js:4:26)
    at /packages/script-runner/lib/script-runner.js:152:2)
    at Module.get_Module._compile (/usr/share/atom/resources/app/src/native-compile-cache.js:106:36)
    at Object.value [as .js] (/usr/share/atom/resources/app/src/compile-cache.js:239:29)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (/app.asar/static/index.js:47:45)
    at require (internal/module.js:20:19)
    at customRequire (/usr/share/atom/resources/app/static/<embedded>:96:26)
    at Package.module.exports.Package.requireMainModule (/usr/share/atom/resources/app/src/package.js:937:35)
    at /usr/share/atom/resources/app/src/package.js:160:34
    at Package.module.exports.Package.measure (/usr/share/atom/resources/app/src/package.js:99:21)
    at Package.module.exports.Package.load (/usr/share/atom/resources/app/src/package.js:145:18)
    at PackageManager.loadAvailablePackage (/usr/share/atom/resources/app/src/package-manager.js:618:16)
    at config.transact (/usr/share/atom/resources/app/src/package-manager.js:531:20)
    at Config.module.exports.Config.transact (/usr/share/atom/resources/app/src/config.js:342:22)
    at PackageManager.loadPackages (/usr/share/atom/resources/app/src/package-manager.js:529:23)
    at /usr/share/atom/resources/app/src/atom-environment.js:832:34


### Non-Core Packages

angularjs 0.5.1 
atom-alignment 0.13.0 
atom-beautify 0.30.9 
atom-bracket-highlight 0.3.2 
atom-brackets-glow 0.6.0 
atom-css-comb 3.3.1 
atom-jade 0.3.0 
atom-jshint 2.0.0 
atom-material-syntax 1.0.7 
atom-material-ui 2.0.6 
atom-monokai 0.10.9 
atom-monokai-dark 0.4.0 
atom-react-autocomplete 0.1.10 
atom-react-native-autocomplete 0.0.27 
atom-terminus 1.1.0 
atom-ternjs 0.18.3 
atom-typescript 11.0.10 
atom-wrap-in-tag 0.6.0 
auto-detect-indentation 1.3.0 
autoclose-html 0.23.0 
autocomplete-javascript 0.0.2 
autocomplete-js-import 1.4.0 
autocomplete-json 5.5.2 
autocomplete-modules 1.8.0 
block-comment-plus 0.5.1 
bracket-padder 0.4.2 
busy 0.7.0 
busy-signal 1.4.3 
change-case 0.6.5 
chester-atom-syntax 0.3.0 
color-picker 2.2.5 
color-tabs 0.1.8 
copy-filename 1.1.0 
define-jump 1.0.1 
disable-keybindings 0.6.0 
editorconfig 2.2.2 
emmet 2.4.3 
emmet-jsx-css-modules 1.0.0 
enhanced-tabs 1.7.0 
es6-javascript 1.0.0 
eval-javascript 1.2.0 
expose 0.15.0 
fancy-bracket-matcher 2.1.0 
file-icons 2.1.14 
foldername-tabs 0.2.4 
git-plus 7.10.0 
goto-definition 1.3.4 
grammar-token-limit 0.1.1 
highlight-line 0.12.0 
highlight-selected 0.13.1 
hyperclick 0.1.5 
intentions 1.1.5 
jade-beautify 0.1.5 
javascript-snippets 1.2.1 
jsformat 0.9.3 
jshint 1.8.6 
language-babel 2.80.3 
language-javascript-jsx 0.3.7 
language-sql-mysql 0.3.1 
linter 2.2.0 
linter-csslint 2.0.0 
linter-eslint 8.4.0 
linter-htmlhint 1.4.0 
linter-jshint 3.1.6 
linter-ui-default 1.6.10 
lucid-tabs 1.1.1 
monokai 0.24.0 
monokai-seti 0.9.0 
new-tab 0.5.3 
nuclide 0.270.0 
oceanic-next 1.0.0 
open-in-browsers 0.0.30 
open-terminal-here 2.3.1 
path-copy 0.14.0 
pigments 0.40.2 
platformio-ide-terminal 2.7.0 
prettier-atom 0.43.1 
pretty-json 1.6.4 
pristine-ui 1.1.11 
project-manager 3.3.5 
pure-syntax 0.3.0 
qolor 0.4.2 
react 0.17.0 
react-es6-snippets 0.3.0 
react-redux-atom-snippets 0.7.2 
react-snippets 1.0.1 
save-session 0.15.7 
script-runner 2.2.1 
seti-ui 1.9.0 
single-click-open 0.2.2 
swackets 0.35.0 
sync-settings 0.8.3 
tab-numbers 0.6.1 
tabasco 0.5.0 
terminal-plus 0.14.5 
todo-show 2.1.0 
tool-bar 1.1.0 
trailing-spaces 0.4.0 
webbox-color 0.5.9 
wrap 0.1.0 
```
And running ``apm -v`` gives me the following
```
apm  1.18.11
npm  3.10.10
node 6.9.5 x64
atom 1.23.1
python 2.7.12
git 2.7.4
```

**Comment by ioquatix** [comment]
Hmm, that's odd. I'll take look.

**Comment by ramblingbarney** [comment]
apm  1.18.8
npm  3.10.10
node 6.9.5 x64
python 2.7.13
git 2.11.0
same issue this morning

**Comment by ioquatix** [comment]
This should be fixed in v2.2.2 which was just released.
